src/documents/tutorial-userInterfaceOptions/UserInterfaceOptions.md

```diff
@@ -35,106 +35,110 @@ These are the basic steps to add UI Options to your website:
 
 ## Add dependencies to the page ##
 
 * the CSS files, and
-* the main Infusion JavaScript file, `infusion_custom.js`.
+* the main Infusion JavaScript file: `infusion-uiOptions.js`.
 
-In the header of your file, link to the CSS files with `<link>` tags (you may have to adjust the paths to reflect where you've saved the Infusion package).
+In the `<head>` of your file, link to the CSS files with `<link>` tags (you may have to adjust the paths to reflect where you've saved the Infusion package). We'll also add a `<script>` tag to link to the UI Options JavaScript.
 
 ```html
-<link rel="stylesheet" type="text/css" href="lib/infusion/src/lib/normalize/css/normalize.css" />
-<link rel="stylesheet" type="text/css" href="lib/infusion/src/framework/core/css/fluid.css" />
-<link rel="stylesheet" type="text/css" href="lib/infusion/src/framework/preferences/css/Enactors.css" />
-<link rel="stylesheet" type="text/css" href="lib/infusion/src/framework/preferences/css/PrefsEditor.css" />
-<link rel="stylesheet" type="text/css" href="lib/infusion/src/framework/preferences/css/SeparatedPanelPrefsEditor.css" />
+<head>
+    <!-- CSS files for UI Options-->
+    <link rel="stylesheet" type="text/css" href="lib/infusion/src/lib/normalize/css/normalize.css" />
+    <link rel="stylesheet" type="text/css" href="lib/infusion/src/framework/core/css/fluid.css" />
+    <link rel="stylesheet" type="text/css" href="lib/infusion/src/framework/preferences/css/Enactors.css" />
+    <link rel="stylesheet" type="text/css" href="lib/infusion/src/framework/preferences/css/PrefsEditor.css" />
+    <link rel="stylesheet" type="text/css" href="lib/infusion/src/framework/preferences/css/SeparatedPanelPrefsEditor.css" />
+
+    <!-- The Infusion Library for UI Options -->
+    <script type="text/javascript" src="lib/infusion/infusion-uiOptions.js"></script>
+</head>
 ```
 
-We'll use the `<script>` tag to link to the Infusion library:
-
-```html
-<!-- The Infusion Library -->
-<script type="text/javascript" src="lib/infusion/infusion-custom.js"></script>
-```
-
-<div class="infusion-docs-note"><strong>Note:</strong> that the <code>infusion-custom.js</code> file is a concatenation of all of the required JavaScript files and will be minified (i.e. all of the whitespace removed) if you've built the minified version. If so, it might be difficult to debug with. If you want to be able to debug the code, you might want to choose the "source" version when you create your Infusion bundle.</div>
+<div class="infusion-docs-note"><strong>Note:</strong> If you are using the minified version of `infusion-uiOptions.js` (i.e. all of the whitespace removed), you can still debug the code using the provided source maps. For more information, visit [the Infusion Release Readme](https://github.com/fluid-project/infusion/blob/master/README.md#source-maps).</div>
```

Comment:
"provided source maps" should be "provided source map". I believe only one source map is provided per build type.